pkg/daemon/file_writers.go

```diff
@@ -60,6 +61,12 @@ func createOrigFile(fromPath, fpath string) error {
 		return writeFileAtomicallyWithDefaults(noOrigFileStampName(fpath), nil)
 	}
 
+	// https://issues.redhat.com/browse/OCPBUGS-11437
```

Comment:
When commenting a bug like this, it means that the code that follows is intended as a workaround, and we would want to remove the code at a later time.

I think that makes sense here, since we eventually want to fix the "orig file" implementation, either separately or eventually as part of layering. Maybe it would be good to mention that here? i.e. this code can be eventually superceded by a proper fix to the underlying problem